src/SFA.DAS.Apprenticeships.Application/Services/ApprenticeshipService.cs

```diff
@@ -127,5 +127,23 @@ public async Task<GetPendingStartDateChangeResponse> GetPendingStartDateChange(G
             var result = await _apiClient.Get<GetPendingStartDateChangeResponse>(new GetPendingStartDateChangeRequest(apprenticeshipKey));
             return result.Body;
         }
-    }
+
+        public async Task ApprovePendingStartDateChange(Guid apprenticeshipKey, string userId)
```

Comment:
We should now be deriving UserId from bearertoken in the apprenticeship inner?

Reply:
If that's the case we'd need to change it across the board, worth creating a ticket to cover that?